SQL/0000-00-00-schema.sql

```diff
@@ -2347,3 +2344,17 @@ CREATE TABLE `issues_watching` (
   KEY `fk_issues_watching_2` (`issueID`),
   CONSTRAINT `fk_issues_watching_1` FOREIGN KEY (`userID`) REFERENCES `users` (`UserID`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+DROP TABLE IF EXISTS `user_psc_rel`;
+CREATE TABLE `user_psc_rel` (
+  `UserID` int(10) unsigned NOT NULL default '0',
+  `CenterID` tinyint(2) unsigned default NULL,
```

Comment:
On mysql 5.7, according to ERROR 1171 (42000): All parts of a PRIMARY KEY must be NOT NULL; if you need NULL in a key, use UNIQUE instead